aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Deploy ASP.NET Core apps to Azure App Service
-author: guardrex
+author: bradygaster
```

Comment:
btw -- This was only done to save U the pings! :smile: lol ... There's no author byline with my name on it. *Please do take the pings! ... They're alllllll yours!* 😀

Reply:
no worries or complaints from me @guardrex - I like being on there so they ping me. this is one of my areas so it makes sense that I get pinged when folks comment here.

Reply:
Of course, by all means. I just wanted to make sure everyone was clear that I don't take credit for things that I don't write or work on. I was only here for the pings and to triage issues. This topic never had an author byline. Since this is your 👶, you might want to put a ...

```
By [Brady Gaster](https://twitter.com/bradygaster)
```

... under the H1.

Reply:
yeah I don't like adding those bylines. I figure use the author metadata.